src/lib/wallet/utils.js

```diff
@@ -279,6 +279,5 @@ export const fromDecimals = (amount, chainOrToken = null) => {
 export const isTransferTx = (txType: string) => /(send|receive|withdraw)(?!.*bridge|pending)/.test(txType)
 
 export const isDuplicateTxError = message =>
-  message
-    .toLowerCase()
-    .search('same nonce|same hash|alreadyknown|already known|feetoolow|nonce is too low|underpriced') >= 0
+  message.toLowerCase().search('same nonce|same hash|alreadyknown|already known|feetoolow|nonce too low|underpriced') >=
```

Comment:
you could reduce amount of variants and use \s* to specify an optional space
```suggestion
  message.search(/(same\s*(nonce|hash)|already\s*known|(fee|nonce)\s*too\s*low|underpriced)/i) >=
```

Reply:
changed